packages/base/src/commands.ts

```diff
@@ -1233,10 +1233,7 @@ export function addCommands(
       }
       const jcadModel = current.context.model;
       const newObject = { ...clipboard, name: newName };
-      jcadModel.syncSelected(
-        { [newObject.name]: { type: 'shape' } },
-        uuid()
-      );
+      jcadModel.syncSelected({ [newObject.name]: { type: 'shape' } }, uuid());
```

Comment:
We should selected **after** adding the object

Reply:
Done :)